components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -1627,6 +1656,20 @@ impl Animatable for MatrixDecomposed2D {
     }
 }
 
+impl ComputeSquaredDistance for MatrixDecomposed2D {
```

Comment:
This implementation can be derived, I think; `Angle` already implements `ComputeSquaredDistance` correctly using radians.